tests/Integration/Database/EloquentBelongsToManyTest.php

```diff
@@ -598,6 +600,7 @@ public function testNoTouchingHappensIfNotConfigured()
         $this->assertNotSame('2017-10-10 10:10:10', $tag->fresh()->updated_at->toDateTimeString());
     }
 
+    /** @group SkipMSSQL */
```

Comment:
We're explicitly skipping some tests on MSSQL because of the following issue: https://github.com/laravel/framework/issues/27778

At a later point we should refactor these maybe to not insert incremental identifiers.